telemetry-core/src/main/java/com/newrelic/telemetry/metrics/MetricBuffer.java

```diff
@@ -44,9 +46,40 @@ public MetricBuffer(Attributes commonAttributes) {
    * @param metric The new {@link Metric} instance to be sent.
    */
   public void addMetric(Metric metric) {
+    System.out.println(metric.getClass());
     metrics.add(metric);
   }
 
+  public void addMetric(Count countMetric) {
+    Map<String, Object> attributes = countMetric.getAttributes();
+
+    ingestWarnings.isValidNumberOfAttributes(attributes, "Metric");
```

Comment:
lets refactor the three ingest warnings checks into one method, so less duplication throughout this class